lnbits/helpers.py

```diff
@@ -160,27 +158,28 @@ def url_for(endpoint: str, external: Optional[bool] = False, **params: Any) -> s
 
 
 def template_renderer(additional_folders: List = []) -> Jinja2Templates:
+
     t = Jinja2Templates(
         loader=jinja2.FileSystemLoader(
             ["lnbits/templates", "lnbits/core/templates", *additional_folders]
         )
     )
 
-    if settings.LNBITS_AD_SPACE:
-        t.env.globals["AD_TITLE"] = settings.LNBITS_AD_SPACE_TITLE
-        t.env.globals["AD_SPACE"] = settings.LNBITS_AD_SPACE
-    t.env.globals["HIDE_API"] = settings.LNBITS_HIDE_API
-    t.env.globals["SITE_TITLE"] = settings.LNBITS_SITE_TITLE
-    t.env.globals["LNBITS_DENOMINATION"] = settings.LNBITS_DENOMINATION
-    t.env.globals["SITE_TAGLINE"] = settings.LNBITS_SITE_TAGLINE
-    t.env.globals["SITE_DESCRIPTION"] = settings.LNBITS_SITE_DESCRIPTION
-    t.env.globals["LNBITS_THEME_OPTIONS"] = settings.LNBITS_THEME_OPTIONS
-    t.env.globals["LNBITS_VERSION"] = settings.LNBITS_COMMIT
+    if settings.lnbits_ad_space:
+        t.env.globals["AD_SPACE"] = settings.lnbits_ad_space
```

Comment:
was this removed on purpose?
`t.env.globals["AD_TITLE"] = settings.LNBITS_AD_SPACE_TITLE`